src/styles/blocks/header.scss

```diff
@@ -0,0 +1,61 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.page-content__header {
+  margin-bottom: 50px;
+}
+
+.logo {
+  width: #{$logoSize};
+  height: #{$logoSize};
+}
```

Comment:
You should move this to separate file, or simply make it element of `header` block and leave it here.